client/web/src/insights/components/form/hooks/useForm.ts

```diff
@@ -262,7 +262,9 @@ export function useForm<FormValues extends object>(props: UseFormProps<FormValue
                 // Hack to focus first invalid input on submit, since we are not using
                 // native behavior in order to avoid poor UX of native validation focus on error
                 // we have to find and focus invalid input by ourselves
-                formElementReference.current?.querySelector<HTMLInputElement>('input:invalid')?.focus()
+                formElementReference.current
+                    ?.querySelector<HTMLInputElement>('input:invalid, textarea:invalid')
```

Comment:
Support textarea invalid case for `repositories` field

Reply:
Is there any reason to not use `:invalid` by itself? Currently it would also not include `<select>`s
```suggestion
                    ?.querySelector<HTMLInputElement>(':invalid')
```

Reply:
@felixfbecker there's also a fieldset element that might be in an invalid state as well (but we can't/don't want to focus invalid fieldset element). Because of this, we have to narrow down invalid element search query to input and textarea.